src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/model/State.java

```diff
@@ -9,26 +9,26 @@
 public enum State {
     @CCD(
         name = "Draft",
-        label = "### Case number: ${[CASE_REFERENCE]}\n ### ${applicant1FirstName} ${applicant1LastName}\n"
+        label = "### #${[CASE_REFERENCE]}\n ### Child's Name: ${childrenFirstName} ${childrenLastName}\n"
```

Comment:
I have kept the placeholder for Children First and Last Name so that whenever we find a solution to this, we'll not have to make changes again to this file